### PR TITLE
No promise, but tests are working - related to issue #2

### PR DESCRIPTION
**Pull request by emettely**
**Is your Pull Request request related to another [issue](https://github.com/bbc/react-transcript-editor/issues) in this repository ?**      
https://github.com/bbc/digital-paper-edit-api/issues/2

**Describe what the PR does**    
Added very crude tests, fixed comments https://github.com/bbc/digital-paper-edit-api/pull/3

**State whether the PR is ready for review or whether it needs extra work**    
Ready for review
Also, this is the other half:
https://github.com/bbc/digital-paper-edit-infrastructure/tree/master/dpe-api/config

## Definitions of Done
- [ ] Runs locally
- [ ] Runs remotely
- [x] Test passes
- [x] Demonstrated
- [x] Deployed to Cosmos on Test and Live (deployed to int and test)
- [ ] Documentation
  - [ ] Developer Documentation - [repo's README|<link>]
  - [ ] Stakeholder Documentation - [Confluence|<link>]
  - [ ] Operational Documentation - [Runbook|<link>]
- [ ] Peer reviewed by:

### CODE REVIEW

**Comment by emettely** [comment]
https://digital-paper-edit-api.int.tools.bbc.co.uk/queue

File: package.json [hunk 1]
```diff
@@ -24,15 +24,28 @@
...
   "dependencies": {
+    "aws-sdk": "^2.482.0",
     "body-parser": "^1.19.0",
     "cuid": "^2.1.6",
+    "dotenv": "^8.0.0",
     "express": "^4.16.4",
-    "express-graphql": "^0.8.0",
     "formidable": "^1.2.1",
-    "graphql": "^14.2.1"
+    "fs": "0.0.1-security"
```
Comment (jamesdools) [inline package.json]: What's this for?

File: tests/MessageBroker.test.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+const chai = require('chai');
```
Comment (jamesdools) [inline tests/MessageBroker.test.js]: can un-capitalise the module name 🤷‍♂

File: package.json (hunk 1 above)
Comment (emettely) [reply package.json]: I think I added it by mistake! Will remove

File: tests/MessageBroker.test.js (hunk 2 above)
Comment (emettely) [reply tests/MessageBroker.test.js]: 👍

**Pull request merged**